### PR TITLE
Add EDAV_info_Categorical_Variables_Chinese_Translation and images

### PR DESCRIPTION
**Pull request by Ch3557**
Add EDAV_info_Categorical_Variables_Chinese_Translation and images

### CODE REVIEW

**Review by yosha-tomar — CHANGES_REQUESTED** [review]
Please change filename to all lowercase characters

See point 1 in section 2.2 here:
https://jtr13.github.io/cc21/github-submission-instructions.html#preparing-your-.rmd-file

**Comment by Ch3557** [comment]
Updated file names to lower cases.

File: edav_info_categorical_variables_Chinese_translation.Rmd [hunk 1]
```diff
@@ -0,0 +1,289 @@
+# Chinese Translation of Bar Chart and Cleveland Dot Plot 
+
+Changhao He
+
+Source: https://edav.info/bar.html<br/>
+        https://edav.info/cleveland.html
+
+## 1.图表：条形图
```
Comment (yosha-tomar) [inline edav_info_categorical_variables_Chinese_translation.Rmd:8]: Please remove the numbers after the hashtag. The hashtags will automatically create the numbered subheadings. Please refer point number 5 here:
https://jtr13.github.io/cc21/tutorial-for-pull-request-mergers.html#check-.rmd-file-contents

This is how it should look:
```suggestion
## 图表：条形图
```
Please change for all following subheadings in the file too.

**Review by yosha-tomar — CHANGES_REQUESTED** [review]
Thank you the update. Please make the following changes as well.

File: edav_info_categorical_variables_Chinese_translation.Rmd (hunk 1 above)
Comment (Ch3557) [reply edav_info_categorical_variables_Chinese_translation.Rmd]: Removed numbers after hashtags.

**Comment by yosha-tomar** [comment]
Thank you for your contribution!

**Pull request merged**